src/screens/EventDetailsScreen/component.js

```diff
@@ -161,7 +166,12 @@ const styles = StyleSheet.create({
     marginTop: 16
   },
   buyButton: {
-    marginTop: 16
+    backgroundColor: whiteColor,
+    paddingVertical: 12,
+    shadowColor,
+    shadowOffset: { width: 0, height: -6 },
```

Comment:
`shadow` properties only work on iOS. You will need to use:
```
<LinearGradient
  colors={[blackThirtyColor, blackZeroColor]}
  style={styles.gradient}
/>
```
to make it work on android.